export/cloud_datastore_admin.py

```diff
@@ -0,0 +1,80 @@
+# Copyright 2018 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Request handler for exporting backups."""
+
+import datetime
+import httplib
+import json
+import logging
+
+from google.appengine.api import app_identity
+from google.appengine.api import urlfetch
+import webapp2
+
+
+class Export(webapp2.RequestHandler):
+
+    def get(self):
```

Comment:
Guard this by a cron header check? See cron.py for how to do this --   `@acl_decorators.can_perform_cron_tasks`

Reply:
I'm having some trouble with this, when I try running the cron job it keeps saying it can't find the module core.domain? I tried putting an __init__.py in the export folder but it seems to say the same thing

Reply:
Interesting, I wouldn't have expected that. I wonder whether it's because app engine is regarding export/ as its own folder that's isolated from the world.

Perhaps take a look at pythonpath in that file to see what's in the pythonpath, and contrast it to what you get when logging pythonpath in a file in core?

If they truly are separated then it's fine to add a decorators.py file in export/ as well.

Reply:
I took a look into the pythonpath, and it does seem like it's different, since the first thing listed is referring to the cloud-datastore-admin app specifically. I went ahead and made a separate acl_decorators.